bindata/deployment/openshift/metallb-openshift.yaml

```diff
@@ -220,7 +220,7 @@ spec:
     spec:
       containers:
         - args:
-            - --enable-webhook
+            - --disable-cert-rotation=true
```

Comment:
Nit on the commit message: we don't use cert-manager in ocp. Afaik it's a custom mechanism.

Reply:
and how the secret is created?

Reply:
I am not 100% sure, but we use a custom `service.beta.openshift.io/inject-cabundle` annotation in ocp which is not cert-manager's one I think.

Reply:
amm right